### PR TITLE
Bn host

### PR DESCRIPTION
**Pull request by Tech-Genius**

### CODE REVIEW

**Review by echefulouis — COMMENTED** [review]
Please resolve conflicts

File: backend/base.html [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
 integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous" />
 <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="{% static 'css/styles/devibeans.min.css' %}">
+<link rel="stylesheet" href="{% static 'css/library_detail.css' %}">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;500&display=swap" rel="stylesheet">
+<script src="{% static 'js/highlight.min.js' %}"></script>
```
Comment (spykelionel) [inline backend/base.html:25]: No need to download. Will get it from a CDN. You can remove this both from static dir

File: backend/base.html [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+<script src="{% static 'js/clipboard.min.js' %}"></script>
+<script>hljs.highlightAll();</script>
```
Comment (spykelionel) [inline backend/base.html:27]: Here too. Remove this line 27

File: backend/library/admin.py [hunk 3]
```diff
@@ -5,3 +5,4 @@
 
 admin.site.register(Comment)
 admin.site.register(Post)
+
```
Comment (spykelionel) [inline backend/library/admin.py:8]: Short version. 

_models = [Post, Comment]

admin.sites.register(_models)
#but all good 👍

File: backend/static/css/login.css [hunk 4]
```diff
@@ -0,0 +1,148 @@
+
+body{
```
Comment (spykelionel) [inline backend/static/css/login.css:2]: Will need to give it a class of lp-body so it doesn't affect other pages. Also in login.html

File: backend/static/css/style.css [hunk 5]
```diff
@@ -0,0 +1,366 @@
+body {    
+    display: flex;
```
Comment (spykelionel) [inline backend/static/css/style.css:2]: Delete this file. All styles already in landing.css.

#duplicate

File: backend/static/css/styles/a11y-dark.min.css [hunk 6]
```diff
@@ -0,0 +1,7 @@
+pre code.hljs{display:block;overflow-x:auto;padding:1em}code.hljs{padding:3px 5px}/*!
```
Comment (spykelionel) [inline backend/static/css/styles/a11y-dark.min.css:1]: Delete this file. Will get it from CDN

**Review by spykelionel — CHANGES_REQUESTED** [review]
Delete all highlight.js files. We'll get it from the cdn

**Review by spykelionel — APPROVED** [review]
- [x] - Delete highlight.js related files. We'll get it from the cdn

**Pull request merged**